argocd/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN curl -sSLf https://github.com/gobuffalo/packr/releases/download/v${PACKR_VER
 
 # Install kubectl
 # NOTE: keep the version synced with https://storage.googleapis.com/kubernetes-release/release/stable.txt
-ENV KUBECTL_VERSION=1.19.7
+ENV KUBECTL_VERSION=1.21.0
```

Comment:
Why not 1.20?

Reply:
I just followed the version [this](https://storage.googleapis.com/kubernetes-release/release/stable.txt) indicates, but 1.20 is better for now.